style.go

```diff
@@ -128,17 +136,19 @@ func NewFill(patternType, fgColor, bgColor string) *Fill {
 }
 
 type Font struct {
-	Size      int
-	Name      string
-	Family    int
-	Charset   int
-	Color     string
-	Bold      bool
-	Italic    bool
-	Underline bool
-}
-
-func NewFont(size int, name string) *Font {
+	Size          float64
```

Comment:
Should this be a float32 or float64?

Reply:
I attempted to use the type as float32, and it is not as succinct as float64. When converting to string from float32 and vice versa, there is an extra step of conversion since the return value of `strconv.ParseFloat()` is float64 anyway.